docs/user-guide/admin-cli.md

```diff
@@ -315,90 +389,95 @@ optional arguments:
 
 #### Add
 Grants datastore access to a vmgroup.
-
-The datastore will be automatically set as "default_datastore" for the vmgroup
-when you grant first datastore access for a vmgroup.
-
+Valid value for "datastore" includes the name of valid datastores in the ESX host , special value "__VM_DS" or "_ALL_DS".
+If "datastore" is set to "__VM_DS", it will create a privilege for VM in this vmgroup to access the "vm_datastore" where VM lives.
```

Comment:
"Create privilege" feels like a slang. I think it's better "When DS is set to _ALL_DS, Access to all DS is l be allowed for vms in vmgroup" 

Same for the next line